lsp-diagnostics.el

```diff
@@ -75,12 +75,13 @@ on top the flycheck face for that error level."
                   "ext:flycheck" (symbol docstring &rest properties))
 (declare-function flycheck-error-new "ext:flycheck" t t)
 (declare-function flycheck-error-message "ext:flycheck" (err) t)
+(declare-function flycheck-error-group "ext:flycheck" (err) t)
 (declare-function flycheck-define-error-level "ext:flycheck" (level &rest properties))
 (declare-function flycheck-buffer "ext:flycheck")
 (declare-function flycheck-valid-checker-p "ext:flycheck")
 (declare-function flycheck-stop "ext:flycheck")
 
-(declare-function lsp-cpp-flycheck-clang-tidy-error-explainer "lsp-cpp")
+(declare-function lsp-cpp-flycheck-clang-tidy-error-explainer "clients/lsp-clangd")
```

Comment:
There should not be `clients/` here, since when package is installed, all of clients directory is flattened